src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dec2Hex.txt

```diff
@@ -177,12 +180,21 @@ Table({Value:"1"},{Value:"0"},{Value:Error({Kind:ErrorKind.Numeric})},{Value:"2"
 >> Dec2Hex([10,11,"12"], [2,5,10])
 Table({Value:"0A"},{Value:"0000B"},{Value:"000000000C"})
 
+>> Dec2Hex(["10","11","12"])
+Table({Value:"A"},{Value:"B"},{Value:"C"})
+
+>> Dec2Hex(["10","11","12"], [2,5,10])
```

Comment:
>["10","11","12"] [](http://example.com/codeflow?start=11&length=16)

Any testcase where first arg is not a valid numeric table type?

Dec2Hex([{A:10}], 3)

Reply:
This kind of test can be found in Dec2HexT_ConsistentOneColumnTableResult.txt